deploy/postgresql-cluster/Chart.yaml

```diff
@@ -10,8 +10,8 @@ version: 0.5.1-beta.0
 # and this value should be consistent with an existing clusterVersion.
 # All supported clusterVersion versions can be viewed through `kubectl get clusterVersion`.
 # The current default value is the highest version of the PostgreSQL (with Patroni HA) supported in KubeBlocks.
-appVersion: "14.7.2"
+appVersion: "14.8.0"
 
 annotations:
-  kubeblocks.io/clusterVersions: "14.7.2,12.14.0"
+  kubeblocks.io/clusterVersions: "14.7.2,12.14.0,12.15.0,14.8.0"
```

Comment:
should older versions be removed?

Reply:
should older version gets removed (as upgraded)?

Reply:
Removed the older versions, only use the latest recommended versions.